TechnicoServer/TechnicoDomain/Services/PropertyItemService.cs

```diff
@@ -0,0 +1,79 @@
+﻿using TechnicoDomain.Dtos;
+using TechnicoDomain.Models;
+using TechnicoDomain.Repositories;
+
+namespace TechnicoDomain.Services;
+
+public class PropertyItemService : IPropertyItemService
+{
+    private readonly IRepository<PropertyItem, long> _repairRepository;
+
+    public PropertyItemService(IRepository<PropertyItem, long> repairRepository)
+    {
+        _repairRepository = repairRepository;
+    }
+
```

Comment:
The CreatePropertyItemAsync and UpdatePropertyItemAsync methods return PropertyItemResponseDto objects in a similar manner. I would suggest to reate a helper method to convert from PropertyItem to PropertyItemResponseDto.